### DISCUSSION TITLE
Display folder % even if to no target describe

### DISCUSSION THREAD

**Discussion by nmathey** [MadeInPierre/finalynx#88] [Ideas]
Hello,

Il serait pratique d'afficher le %/ratio de chaque 'Folder' mais si il n'y a pas de 'Target' associé.
Cela nous permettra de quand même avoir l'info de ce qu'il représente dans le portefeuille.

**Comment by MadeInPierre** [comment]
Hello, ça pourrait être bien en effet, mais cette règle ne s'applique sûrement pas à toutes les lignes. Par exemple, dans cette partie de ma config perso : 

![image](https://github.com/MadeInPierre/finalynx/assets/12670280/0860ea86-ff47-4082-a69a-c42374a004a8)

Si je n'avais pas mis de Target, il y aurait des pourcentages affichés et c'est un peu inutile... Tu verrais un moyen pour savoir quand afficher ces pourcentages ?

**Reply by nmathey** [reply]
Hello,

Peut-être en laissant le choix via un attribut comme "Target" car c'est vrai cela dépend comment chacun structure son portefeuille Finalyx.
Après même dans ton cas cela te donnera une idée de la répartition de tes enveloppes à chaque niveau à un instant T ^^

**Reply by MadeInPierre** [reply]
En vrai on pourrait rendre le premier argument de `TargetRatio` optionnel, et si quelqu'un déclare :

```python
Line(..., target=TargetRatio())
```

Alors on affiche le pourcentage actuel de l'investissement sans code couleur et tout. 

Ça te paraît logique/intuitif ?

**Comment by nmathey** [comment]
Oui ca peut le faire et tant que c'est documenté dans ce sens la logique est bonne :)

**Reply by nmathey** [reply]
En fait la demande s'appliquerait également (voir plus) au niveau des '`Folder`': sans avoir à donner de target ce serait intéressant d'afficher le pourcentage que représente les '`children`' dans certain cas comme ici par exemple:

<img width="332" alt="Capture d’écran 2023-05-29 à 16 24 40" src="https://github.com/MadeInPierre/finalynx/assets/20896232/8ef66069-c154-4344-8fb0-0bf86b510ff9">

je fais une issue dans ce sens pour la rajouter à la ToDo list ?

**Reply by MadeInPierre** [reply]
Créé #110, je devrais pouvoir la faire bientôt :crossed_fingers: On peut essayer d'afficher le pourcentage pour tout le monde par défaut (sans faire `target=TargetRatio()`), et si c'est chiant on changera la règle mais ça vaut le coup d'essayer ^^
